assets/js/components/NotFound/NotFound.stories.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import NotFound from '.';
+
+export default {
+  title: 'NotFound',
+  component: NotFound,
+  args: { buttonText: 'Go back home' },
```

Comment:
```suggestion
  args: { buttonText: 'Go back home', onNavigate: () => {} },
```

Reply:
Nice catch thanks, as the story was written before the refactor xD